Price Calculator/DiscountInformation.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Price_Calculator
+{
+    public class DiscountInformation
+    {
+
+        public enum DiscountType
+        {
+            AfterTax,
+            BeforeTax
+        }
+        public enum DiscountMethod
+        {
+            Additive,
+            Multiplicative
+        }
+        public static decimal Discount { get; set; }
+        public static decimal UPCDiscount { get; set; } = 0;  // Percentage 
+        public static int DiscountedUPC { get; set; } = 0;  // applied for this UPC only
+
```

Comment:
it's better to extract these properties into new type like UPCDiscount, and have the value and the code of UPC, and here add property of the new type, 
similar to what you did in Expense